modules/distributor/forwarder/manager.go

```diff
@@ -21,10 +22,11 @@ const (
 )
 
 type Overrides interface {
-	TenantIDs() []string
```

Comment:
This method wasn't used, I've removed it and added a type assertion to avoid drift.